internal/api/client.go

```diff
@@ -6,7 +6,7 @@ import (
 
 // client is a client for the test splitter API.
 // It contains the organization slug, server base URL, and an HTTP client.
-type client struct {
+type Client struct {
```

Comment:
I make this public because I need to pass this type on one of the function in the `main` package.